### PR TITLE
Prepare the v2.5.14-rc2 release (final RC)

### PR DESCRIPTION
**Pull request by maxsokolovsky**

### CODE REVIEW

File: package/Dockerfile [hunk 1]
```diff
@@ -113,7 +113,7 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     chmod +x /usr/bin/tini /usr/bin/telemetry /usr/bin/k3s /usr/bin/kubectl /usr/bin/channelserver && \
     mkdir -p /var/lib/rancher-data/driver-metadata
 
-ENV CATTLE_UI_VERSION 2.5.14-rc1
+ENV CATTLE_UI_VERSION 2.5.14-rc2
```
[the remark this answers is not in the document]
Comment (cbron) [reply package/Dockerfile:116]: You should also double check on the last images.txt, for anything with RC in it's image name. And for any non-release branches or versions of charts/system-charts/kdm/rke etc... Check go.mod too.

[the remark this answers is not in the document]
Comment (maxsokolovsky) [reply package/Dockerfile:116]: Makes sense, but also `rancher/ui v2.5.14-rc2` has been published today. Should it be tested?

Before we remove the `rc`, we must ensure `rancher/ui v2.5.14` exists, right? It does not exist now...

[the remark this answers is not in the document]
Comment (cbron) [reply package/Dockerfile:116]: Yes, you need to ask UI team for their final tag.

[the remark this answers is not in the document]
Comment (maxsokolovsky) [reply package/Dockerfile]: Updated the final UI tag.
Checked non-release branches or versions of charts/system-charts/kdm/rke - it's `release-v2.5` everywhere.
Not sure where image.txt file is though...

**Review by cbron — APPROVED** [review]

**Pull request merged**